### ISSUE TITLE
[BUG] Error when long pressing the on/off button E1743

### ISSUE THREAD

**Issue by duceduc** [xaviml/controllerx#512]
# Bug report

## Description
I see the blow error in log whenever I long press either the 'on' or 'off' button. The on/off state and dimming seem to be working fine.

## Additional information

- Devices involved:
  - Model: Ikea E1743
- Integration: Choose from `z2m`
- AppDaemon version: [ v0.9.0 ]
- ControllerX version: [ v4.21.0b2 ]
- HACS version (if installed from there): [ v1.25.5 ]
- Home Assistant Core version: [ v2022.6.5 ]

## AppDaemon app configuration

```
controller config:
    shower_room:
      module: controllerx
      class: E1743Z2MLightController
      controller: shower_room
      integration:
        name: z2m
        listen_to: mqtt
      light:
        name: shower
        mode: mqtt
```

## Logs

```
Zigbee2MQTT:error 2022-06-14 14:58:31: No converter available for 'color_temp_move' ("stop")
```

**Comment by xaviml** [comment]
Hi @duceduc ,

Probably, your light does not support "color_temp". When release is fired, 2 mqtt messages are sent, a brightness and color temp stop. This is because it is not straight forward to know which hold action was fired (brightness or color temp). So, to be safe, it sends both messages. This does not affect the functionality, but it does show error in Zigbee2MQTT.

I will explore a bit further to fix this and smartly know which attribute was the one holding.

Regards,
Xavi M.

**Comment by duceduc** [comment]
Yes, you are right. My lights doesn't support color.

**Comment by xaviml** [comment]
Then, it is a expected bug (maybe a feature? :P haha). But I will leave this issue open to explore the options.

Thank you for reporting it!

**Comment by duceduc** [comment]
Yea. Not really a bug.

**Comment by xaviml** [comment]
Hi @duceduc ,

I fixed this in #513. I will be releasing the fix on the final 4.21.0 version some days from now.

Regards,
Xavi M.
